dotCMS/src/integration-test/java/com/dotmarketing/portlets/contentlet/transform/ContentletTransformerTest.java

```diff
@@ -235,8 +299,251 @@ public void Test_Constant_fields_are_in_map() throws Exception {
+        map.put(Contentlet.BASE_TYPE_KEY, BaseContentType.FILEASSET);
+        map.put(META_DATA_FIELD, ContentletCache.CACHED_METADATA);
+
+        final ImageField field = mock(ImageField.class);
+        when(field.variable()).thenReturn("imageVar");
+        final Optional<Field> titleImage = Optional.of(field);
+        map.put(Contentlet.TITLE_IMAGE_KEY, titleImage);
+        map.put(Contentlet.HAS_TITLE_IMAGE_KEY, true);
+
+        when(contentlet.getMap()).thenReturn(map);
+
+        when(identifierAPI.find(identifier)).thenReturn(identifierObject);
+
+        final Set<TransformOptions> options1 = EnumSet.of(
+                INC_COMMON_PROPS, INC_VERSION_INFO
+        );
+
+        final FileAsset fileAsset = mock(FileAsset.class);
+        when(fileAsset.getIdentifier()).thenReturn(identifier);
+        when(fileAsset.getContentType()).thenReturn(fileAssetContentType);
+        when(fileAsset.getContentTypeId()).thenReturn(anyFileAssetSubType);
+        when(fileAsset.getMimeType()).thenReturn(mimeType);
+        when(fileAsset.getFileName()).thenReturn(fileName);
+        when(fileAsset.getFileSize()).thenReturn(fileSize);
+        when(fileAsset.getUnderlyingFileName()).thenReturn(underlyingFileName);
+        when(fileAsset.getWidth()).thenReturn(width);
+        when(fileAsset.getHeight()).thenReturn(height);
+        when(fileAsset.getMetaData()).thenReturn("Meta");
+
+        when(fileAssetAPI.fromContentlet(any(Contentlet.class))).thenReturn(fileAsset);
+
+        final List<Map<String, Object>> maps = new DotTransformerImpl(
+                singletonList(contentlet),
+                resolver,
+                options1, user).toMaps();
+
+        final Map<String, Object> mapView = maps.get(0);
+
+        final String returnedMimeType = (String)mapView.get(MIMETYPE_FIELD);
+        final String title = (String)mapView.get(TITLE_FIELD);
+        final String returnedIdentifier = (String)mapView.get(IDENTIFIER_KEY);
+        final String parent = (String)mapView.get("parent");
+        final String extension = (String)mapView.get("extension");
+        final Integer returnedWidth = (Integer)mapView.get("width");
+        final Integer returnedHeight = (Integer)mapView.get("height");
+        final String meta = (String)mapView.get(META_DATA_FIELD);
+        final String icon = (String)mapView.get("__icon__");
+        final String returnedUnderlyingFileName = (String)mapView.get(UNDERLYING_FILENAME);
+
+        //FLAG Alias is OFF
+        Assert.assertFalse(mapView.containsKey("fileName"));
+        Assert.assertFalse(mapView.containsKey("fileSize"));
+        //Flag Metadata is OFF
+
+        Assert.assertEquals(identifier, returnedIdentifier);
+        Assert.assertEquals(urlExpected, title);
+        Assert.assertEquals(mimeType, returnedMimeType);
+        Assert.assertEquals("png",extension );
+        Assert.assertEquals(StringPool.BLANK, parent);
+        Assert.assertEquals(height, returnedHeight.intValue());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Assert.assertEquals' due to existing static import 'org.junit.Assert.assertEquals'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5596402)